development/tools/cmd/synchronizer/main.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	sc "github.com/kyma-project/test-infra/development/tools/cmd/synchronizer/syncomponent"
+	t "github.com/kyma-project/test-infra/development/tools/cmd/synchronizer/tools"
+	"github.com/pkg/errors"
+)
+
+const (
+	envKymaProjectDir  = "KYMA_PROJECT_DIR"
+	slackClientToken   = "SLACK_CLIENT_TOKEN"
+	slackClientChannel = "STABILITY_SLACK_CLIENT_CHANNEL_ID"
+	outOfDate          = "OUT_OF_DATE_DAYS"
+	pathToVersionFile  = "Makefile"
+	varsionPathCommand = "version-path"
```

Comment:
version-path is not informative at all IMO. My proposal:
`used-by-charts`, `referenced-by-charts`, `path-to-referenced-charts`

Reply:
this comment first should be add to this [PR](https://github.com/kyma-project/kyma/pull/2850/files)

Reply:
from yours proposal IMO `path-to-referenced-charts` is the most fitting one